app/models/github_callback.rb

```diff
@@ -0,0 +1,11 @@
+class GithubCallback
+  include ActiveModel::Model
+
+  def initialize(event)
+    if event['ref'] == 'refs/heads/master'
+      repo = event['repository']['name']
+      files = event['commits'].map { |commit| commit['added'] + commit['modified'] + commit['removed'] }.flatten.uniq
+      Github.update_lessons_from_github(repo, files)
```

Comment:
This makes sense to me, though I realized it makes the unit test much less pretty - because to test it the way I rewrote it I have to pass in a full fake webhook payload rather than the repo and files. I'll upload the new version.